.ruby-version

```diff
@@ -0,0 +1 @@
+3.4.2
```

Comment:
I don't think we want to ship .ruby_version file with this gem. This file enforces the specified version. We already have gemspec file with open ended ruby requirement.